blockdev/src/blockdev.rs

```diff
@@ -181,6 +181,14 @@ pub fn partitions_of(dev: &Utf8Path) -> Result<PartitionTable> {
 
 pub struct LoopbackDevice {
     pub dev: Option<Utf8PathBuf>,
+    // Handle to the cleanup helper process
+    cleanup_handle: Option<LoopbackCleanupHandle>,
+}
+
+/// Handle to manage the cleanup helper process for loopback devices
+struct LoopbackCleanupHandle {
+    /// Process ID of the cleanup helper
+    helper_pid: u32,
```

Comment:
This should actually hold the [Child](https://doc.rust-lang.org/stable/std/process/struct.Child.html) instance. And on `Drop` we should send it `SIGTERM` - note *not* [kill](https://doc.rust-lang.org/stable/std/process/struct.Child.html#method.kill) but [rustix kill](https://docs.rs/rustix/latest/rustix/process/fn.kill_process.html) with `SIGTERM`.